MAUI/Cartesian-Charts/Axis/Ticklines.md

```diff
@@ -50,6 +50,42 @@ chart.YAxes.Add(secondaryAxis);
 
 {% endtabs %}
 
+## Positioning the ticklines
+
+The positioning of tick lines inside or outside the chart area can be controlled using the [TickPosition]() property. By default, the tick lines are positioned `AxisElementPosition.Outside` the chart area. 
+
+{% tabs %}
+
+{% highlight xaml %}
+
+<chart:SfCartesianChart>
+    . . .  
+    <chart:SfCartesianChart.YAxes>
```

Comment:
Add x axis code as we discussed for cartesian chart

Reply:
Updated the code snippet